apps/pipedrive-hello-world-glitch/README.md

```diff
@@ -0,0 +1,12 @@
+# Sample App for Pipedrive Marketplace
+
+A very basic example of how to:
+
+- Authorize with OAuth using [passportjs](http://www.passportjs.org/).
+- Get all open deals from Pipedrive.
+- Mark selected deal randomly as won or lost.
+
+## Installation and usage for Glitch Apps
+
+- In the `config.js` file, fill in the `clientID` and `clientSecret` values. You will obtain that once you create an App via the Marketplace Manager.
+- Ensure that the app is running by checking the _Logs_ panel in the footer.
```

Comment:
I'd keep original Installation and usage for local usage:

    Create a test app in [Developer Hub](https://app.pipedrive.com/developer-hub) with the following callback URL: http://localhost:3000/auth/pipedrive/callback
    Run npm install
    Rename config.example.js to config.js and edit the clientID and clientSecret values
    Run node index.js and open http://localhost:3000/

Reply:
Then it is more moving FiftyFifty to example Apps, but I am not against it, I will make the change